submissions/motunrayoIlawole/php/list-view.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+$customers_data;
```

Comment:
I don't think you can declare a variable like this in PHP without a value. Kindly do this instead `$customers_data = '';`